R/pull_GA_data.R

```diff
@@ -2,19 +2,20 @@
 #grab several important applications
 library(googleAnalyticsR)
 library(googleAuthR)
-library(tidyverse)
-gar_auth_service('~/.vizlab/VIZLAB-a48f4107248c.json')
+library(dplyr)
+library(arrow)
+gar_set_client(json = Sys.getenv('GA_CLIENTID_FILE'), 
+               scopes = "https://www.googleapis.com/auth/analytics.readonly")
+gar_auth_service(json_file = Sys.getenv('GA_AUTH_FILE'))
```

Comment:
These two lines will need to be run manually to run this locally (unless you also set up the appropriate environment variables)